### ISSUE TITLE
unable to create instance with open virtual switch - ubuntu 14.04 - ganeti 2.9.3

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#860]
Originally reported of Google Code with ID 804.

```
What software version are you running? Please provide the output of "gnt-
cluster --version", "gnt-cluster version", and "hspace --version".

<b>What distribution are you using?</b>

Ubuntu 14.04

<b>What steps will reproduce the problem?</b>
1. create a cluster
gnt-cluster init --master-netdev=lan1 --vg-name ganeti --enabled-hypervisors=kvm -N link=lan1,mode=openvswitch -H kvm:kernel_path="",initrd_path="",vnc_bind_address=0.0.0.0,serial_console=true,keymap=de_DE -D drbd:metavg=ganeti -B vcpus=2,memory=512M --enabled-disk-templates plain,drbd --secondary-ip 192.168.21.100 control.gnt

2. add a network
gnt-network add --network=10.100.21.0/24 --gateway=10.100.21.1 --no-conflicts-check server

gnt-network connect server openvswitch lan1

3. create a vm
gnt-instance add -s 30G --net 0:network=server,ip=10.100.21.202,vlan=21 -B memory=2G -o debootstrap+default -n gnt01 --no-start --no-install test

<b>What is the expected output? What do you see instead?</b>

	
root@gnt01:/etc/ganeti# gnt-instance add -s 30G --net 0:network=server,ip=10.100.21.202 -B memory=2G -o debootstrap+default -n gnt01 --no-start --no-install test                    
Thu Apr 17 14:59:09 2014  - INFO: NIC/0 inherits netparams ['lan1','openvswitch']
Thu Apr 17 14:59:10 2014 * disk 0, size 30.0G
Thu Apr 17 14:59:10 2014 * creating instance disks...
Thu Apr 17 14:59:11 2014 adding instance test to cluster config
Unhandled Ganeti error: NIC mode 'openvswitch' not handled


<b>Please provide any additional information below.</b>

the command create the lvm disks and shows the instance (gnt-instance list)
after a service ganeti restart the instance is gone and gnt-cluster verify shows a unkown volume

```
Originally added on 2014-04-17 13:37:52 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
ovs-vsctl show
bc454b96-2677-4a91-9d1a-b784d7d265af
    Bridge "drbd1"
        Port "bond-drbd1"
            Interface "em4"
            Interface "em3"
        Port "drbd1"
            Interface "drbd1"
                type: internal
    Bridge "lan1"
        Port "tap0"
            Interface "tap0"
        Port "lan1"
            Interface "lan1"
                type: internal
        Port "em2"
            Interface "em2"
    ovs_version: "2.0.1"

my ovs configuration
```
Originally added on 2014-04-17 13:50:59 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2014-05-20 12:31:56 +0000 UTC.

Added Labels: Type-Defect Priority-Medium
Changed State: Accepted
Added to Milestone: Release2.13

**Comment by Ganeti-Issues-Migrator** [comment]
```
hi, i send a patch that fix this bug.

http://git.ganeti.org/?p=ganeti.git;a=commitdiff;h=95ec6e958aa387813a325e772d825fd5ee34da6a
```
Originally added on 2014-05-21 11:05:47 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2014-05-21 14:19:18 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released as part of 2.14.0
```
Originally added on 2015-06-02 10:37:02 +0000 UTC.

Changed State: Released
